include/deal.II/grid/grid_generator.h

```diff
@@ -1535,6 +1535,11 @@ namespace GridGenerator
    * parallel::distributed::Triangulation, as well as when the input
    * Triangulation contains hanging nodes.
    *
+   * @param[in] in_tria is the base input for a new flat triangulation.
+   * @param[out] out_tria the desired flattened triangulation constructed from the in_tria.
+   * be built to consist of the cells that correspond to the (selected portion
```

Comment:
I think you are missing the first part of this sentence. Could you please add the text?